timeless/poster/tasks.py

```diff
@@ -59,6 +59,23 @@ def sync_customers():
         )
 
 
+@shared_task
+def sync_locations():
+    """
+    Periodic task for fetching and saving location from Poster
+    """
+    for poster_location in __poster_api().locations():
+        location = DB.session(Location).query.filter_by(
```

Comment:
@vladarefiev Can't we use ``Location.query(...)`` instead of ``DB.session(Location)``? I've seen that ``query(...)`` is mostly used aroung the project :D

Reply:
@amihaiemil yes, i can do it here.